### PR TITLE
[ xray ] Fix notes warning regarding master key

### PR DESCRIPTION
**Pull request by lindhe**
If `.Values.xray.masterKeySecretName` was set, `.Values.xray.masterKey`
is still set to `FFFF...` and causes NOTES to print a warning about it.
This commit fixes that, so the incorrect warning is not printed.

#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Instead of setting `xray.masterKey`, I'm using `xray.masterKeySecretName`, yet the following warning was printed:

```
************************************* WARNING *****************************************
* Your Xray master key is still set to the provided example:                          *
* xray.masterKey=FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF     *
*                                                                                     *
* You should change this to your own generated key:                                   *
* $ export MASTER_KEY=$(openssl rand -hex 32)                                         *
* $ echo ${MASTER_KEY}                                                                *
*                                                                                     *
* Pass the created master key to helm with '--set xray.masterKey=${MASTER_KEY}'       *
***************************************************************************************
```

### CODE REVIEW

File: stable/xray/templates/NOTES.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Congratulations! JFrog Xray services are deployed!
 
-{{- if eq .Values.xray.masterKey "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF" }}
+{{- if and (eq .Values.xray.masterKey "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF") .Values.xray.masterKeySecretName }}
```
Comment (chukka) [inline stable/xray/templates/NOTES.txt:3]: @lindhe  I think it should be  `(not .Values.xray.masterKeySecretName)` right ?

Comment (lindhe) [reply stable/xray/templates/NOTES.txt:3]: Certainly. My fault for throwing together this minutes before leaving for the day. Feel free to change it if you want, or I'll do it when I find time. Likely in a few hours.

Comment (chukka) [reply stable/xray/templates/NOTES.txt:3]: Thanks, Please update the PR when you get a chance :)

Comment (lindhe) [reply stable/xray/templates/NOTES.txt]: I think I was able to do it from my phone. Unless I screwed up, I feel very cool.

**Pull request merged**